test/component-limit.js

```diff
@@ -58,6 +59,17 @@ eslintTester.run('component-limit', rule, {
             code: 'var app = angular.module(""); app.animation().animation();',
             errors: [{message: 'There may be at most 1 AngularJS component per file, but found 2'}]
         },
+        // directive
```

Comment:
Missed changing this comment to component.  Kind of new to github, not sure what the right way to correct something like this is mid pull request.